html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Form</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get">
+    <p>Email</p>
```

Comment:
This should be a `label` element. Make sure you configure it correctly, as described in the documentation.